.travis.yml

```diff
@@ -19,8 +19,8 @@ before_script:
   - curl -sSL https://github.com/maidsafe/QA/raw/master/travis/cargo_install.sh > cargo_install.sh
   - bash cargo_install.sh cargo-prune;
   - if [[ $TRAVIS_RUST_VERSION =~ ^nightly.* && $TRAVIS_OS_NAME == linux ]]; then
-      bash cargo_install.sh rustfmt-nightly 0.8.2;
```

Comment:
Can we move these version strings into variables like in our other CI scripts? The rest looks great 👍

Reply:
Will do!